.gitignore

```diff
@@ -29,6 +29,7 @@ o_o_standalone_config_IncludeFileDirectiveTest_part.json
 .venv/
 _site/
 build/
+!/src/build/
```

Comment:
What does this do?

Reply:
The templates reside under /src/build/ (as was suggested by your friend in Berlin). The line above this causes all build directories to be ignored by git so this line is needed to include the build directory under src in version control.

Reply:
Thanks!